### ISSUE TITLE
Partial use when using fqn in no namespaced script

### ISSUE THREAD

**Issue by CZechBoY** [slevomat/coding-standard#459]
Hi,
for this code
```php
<?php declare(strict_types = 1);

require __DIR__ . '/../vendor/autoload.php';
$configurator = new Nette\Configurstor;
```
cs is complaining about partial use. But there is no `use` statement, there is only no namespace.

It can be fixed by typing `\Nette\Configurator' or by import.

Is it bug or feature?

**Comment by kukulich** [comment]
@ondrejmirtes Is it a bug or feature? :)

**Comment by ondrejmirtes** [comment]
Hard to say :) 🤷‍♂️ I'd say that for the sake of consistency, you should have "\Nette\Configurator" everywhere, right? What do you think, @VasekPurchart?

**Comment by kukulich** [comment]
Thanks @ondrejmirtes and @VasekPurchart
